packages/flutter_tools/bin/macos_build_flutter_assets.sh

```diff
@@ -39,12 +39,6 @@ if [[ -n "$TRACK_WIDGET_CREATION" ]]; then
   track_widget_creation_flag="--track-widget-creation"
 fi
 
-# Copy the framework and handle local engine builds.
```

Comment:
What is replacing this?

Reply:
The UnpackMacos target will ensure that the contents of `macos/Flutter/ephemeral/*` are up to date with whatever is in the cache/build mode.

Reply:
Ah, I was confused because that's one of the things that's still around from earlier.

Since we'll be relying on it now, could you fix the output list there as part of this PR? The header list is out of date (https://github.com/flutter/engine/blob/master/shell/platform/darwin/macos/framework/Headers/FlutterMacOS.h), and s/info.plist/Info.plist/